src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/PostgresRepository.cs

```diff
@@ -23,6 +23,9 @@ public class PostgresRepository : IPostgresRepository
         private readonly ILogger _logger;
         private readonly string insertEventSql = "call events.insert_event(@id, @source, @subject, @type, @cloudevent)";
         private readonly string getEventSql = "select events.get(@_subject, @_after, @_from, @_to, @_type, @_source)";
+        private readonly string insertSubscriptionSql = "call events.insert_subcsription(@sourcefilter, @subjectfilter, @typefilter, @consumer, @endpointurl, @createdby, @validated, @v_id)";
```

Comment:
I'm guessing v_id is the out parameter.. is this a naming convention? The parameter name could be more self explanatory if we are not bound by a naming convention

Reply:
renamed. I dont belive it is a standard. just used in an example how to do it.